Source/Blazorise/Components/Bar/BarDropdown.razor.cs

```diff
@@ -48,7 +48,10 @@ protected override void BuildClasses( ClassBuilder builder )
     /// <inheritdoc/>
     protected override Task OnInitializedAsync()
     {
-        ParentBarItem?.NotifyBarDropdownInitialized( this );
+        if ( ParentBarDropdown is null ) //BarItem tracks the first direct menu. Ignore submenus
```

Comment:
For clarity, maybe have `ParentBarDropdown is null` under the getter property named `IsUnderFirstMenu,` or similar.